src/features/auth/hooks/useHandleForbidden.ts

```diff
@@ -0,0 +1,21 @@
+import { AxiosError } from 'axios'
+import { useRouter } from 'next/router'
+import { noAuthRoute } from '@features/auth/utils/noAuthRoute'
+import { StatusCode } from '@utils/StatusCode'
+
+export const useHandleForbidden = (error?: AxiosError) => {
+  const router = useRouter()
+
+  if (!error) return
+
+  const statusCodesMeaningForbidden = [
+    StatusCode.FORBIDDEN,
+    StatusCode.UNAUTHORIZED
+  ]
+
+  const statusCode = Number(error.status)
```

Comment:
Since you always get strings from axios, then your enum should have strings so you don't have to typecast every time.